test/test_autotag.py

```diff
@@ -940,13 +940,13 @@ class EnumTest(_common.TestCase):
     Test Enum Subclasses defined in beets.util.enumeration
     """
     def test_ordered_enum(self):
-        OrderedEnumTest = match.OrderedEnum('OrderedEnumTest', ['a', 'b', 'c'])
-        self.assertLess(OrderedEnumTest.a, OrderedEnumTest.b)
-        self.assertLess(OrderedEnumTest.a, OrderedEnumTest.c)
-        self.assertLess(OrderedEnumTest.b, OrderedEnumTest.c)
-        self.assertGreater(OrderedEnumTest.b, OrderedEnumTest.a)
-        self.assertGreater(OrderedEnumTest.c, OrderedEnumTest.a)
-        self.assertGreater(OrderedEnumTest.c, OrderedEnumTest.b)
+        ordered_enum = match.OrderedEnum('OrderedEnumTest', ['a', 'b', 'c'])
+        self.assertLess(ordered_enum.a, ordered_enum.b)
+        self.assertLess(ordered_enum.a, ordered_enum.c)
+        self.assertLess(ordered_enum.b, ordered_enum.c)
+        self.assertGreater(ordered_enum.b, ordered_enum.a)
+        self.assertGreater(ordered_enum.c, ordered_enum.a)
+        self.assertGreater(ordered_enum.c, ordered_enum.b)
```

Comment:
Yeah, this is also kind of present in [`test/test_smartplaylist.py`]:

```python
sort = FixedFieldSort  # short cut since we're only dealing with this
asseq(sorts["no_sort"], NullSort())
asseq(sorts["one_sort"], sort(u'year'))
asseq(sorts["only_empty_sorts"], None)
asseq(sorts["one_non_empty_sort"], sort(u'year'))
asseq(sorts["multiple_sorts"],
    MultipleSort([sort('year'), sort(u'genre', False)]))
asseq(sorts["mixed"],
    MultipleSort([sort('year'), sort(u'genre'), sort(u'id', False)]))
```

It was originally named `S` instead of `sort`.